Add how to install warp

maxtortime opened the pull request:
#77 
I want to contribute this project but I don't know how to install warp.
Therefore, @mingrammer taught me instructions.

Thanks and I added instructions for installing warp.

Regards,
Taehwan Kim

maxtortime committed:
Add how to install warp

maxtortime commented:
Sorry for I'm newbie of RST, So some of syntax will be wrong.. 😭

qkraudghgh commented:
Thank you for your Contribute.
but we have contribute rule. We want follow this rule plz.
https://github.com/SaturDJang/warp/blob/master/CONTRIBUTING.md

maxtortime committed:
Add more instructions for installing

(#77) Documentation

    A SHORT SUMMARY

    1. Add link for installing docker
2. Commands of docker to create warp container and build.
